opbeat.go

```diff
@@ -161,7 +149,7 @@ func NewFromEnvironment() *Opbeat {
 	timeout := os.Getenv("OPBEAT_TIMEOUT")
 	if len(timeout) > 0 {
 		timeoutSec, err := strconv.Atoi(timeout)
-		if err != nil {
+		if err != nil && opbeat.Logger != nil {
```

Comment:
I say let's not for now. If we want to be able to use `Printf`, `Print` and `Println` things get pretty complicated.